website/content/docs/job-specification/artifact.mdx

```diff
@@ -244,6 +244,12 @@ artifact {
 }
 ```
 
+## Environment
+
+Starting with Nomad 1.5.0, the `artifact` downloader by default does not have access to the environment variables
```

Comment:
```suggestion
The `artifact` downloader by default does not have access to the environment variables
```

With versioned docs we don't need to reference specific versions on the docs anymore, just make sure this gets backported to `release/1.5.x` only.